Assets/Scripts/Health.cs

```diff
@@ -0,0 +1,33 @@
+﻿using UnityEngine;
+
+namespace Assets.Scripts
+{
+    public class Health : MonoBehaviour
+    {
+        public int curHealth = 0;
```

Comment:
curHealth should not be inited to 0. it is inited in Start, so 0 is just confusing